### PR TITLE
Add R Learning Resources

### PR DESCRIPTION
**Pull request by iamericfletcher**
**https://github.com/iamericfletcher/awesome-r-learning-resources**

This list is a collection of resources related to R, a programming language and environment for statistical computing and graphics. It is meant to help users from all skill levels and backgrounds deepen their understanding of R, resulting in a more knowledgable programming population that benefits everybody involved.

Reviewed PRs: [#1857 ](https://github.com/sindresorhus/awesome/pull/1857) [#1848](https://github.com/sindresorhus/awesome/pull/1848)

The link in the `files changed` section should be replaced with the new address at the top of this pull request.

### CODE REVIEW

**Comment by iamericfletcher** [comment]
I'd love to add a logo. Outside of the social preview image I have assigned, I am not sure where I would add the logo? Also, I am not sure what you mean when you say examples for Africa, Asia, and Australia. Could you provide a little more information on that? I will update my repo ASAP to align with your suggestions. Thank you! Eric

**Comment by sindresorhus** [comment]
Thanks for making an Awesome list! 🙌

It looks like you didn't read the [guidelines](https://github.com/sindresorhus/awesome/blob/master/pull_request_template.md) closely enough. I noticed multiple things that are not followed. Try going through the list point for point to ensure you follow it. I spent a lot of time creating the guidelines so I wouldn't have to comment on common mistakes, and rather spend my time improving Awesome.

**Comment by RayBB** [comment]
@iamericfletcher here are some issues:
* You have to review at least 2 other open pull requests. we'll know from looking [here](https://github.com/sindresorhus/awesome/pulls?q=commenter%3Aiamericfletcher+)
* Run awesome-lint on your list and fix the reported issues.  There are currently many issues when I run it. You can make this happen automatically by following [these](https://github.com/sindresorhus/awesome-lint/pull/116) steps.

**Comment by iamericfletcher** [comment]
Thank you very much @RayBB for your time in reviewing my PR. I am trying hard to follow the instructions for awesome-lint, but I am having trouble. I open the README.md in my terminal, add `$ npm install awesome-lint` to the Xcode window that pops up, save and push to main Sadly, I am not seeing anything. I am new to this, so any help would be much appreciated. Thank you. Eric

**Comment by RayBB** [comment]
@iamericfletcher 

1. Open your terminal
2. Navigate to your github repo for awesome R
3. Run `npx awesome-lint` 

Alternatively, you can have github run this for you against any PR you open on your own repo by adding [this](https://github.com/RayBB/awesome-social-enterprise/blob/main/.github/workflows/main.yml) file.

**Comment by iamericfletcher** [comment]
@RayBB I have followed your instruction using my Mac with the following result:

```
Erics-MBP:r-learning-resources ericfletcher$ npx awesome-lint
-bash: run: command not found
```

**Comment by RayBB** [comment]
@iamericfletcher your options are:
1. Setup GitHub actions like [this](https://github.com/RayBB/awesome-social-enterprise/blob/main/.github/workflows/main.yml). [Here](https://youtu.be/F3wZTDmHCFA) is a YT video that explains GitHub Actions.
2. Install [NPM](https://www.npmjs.com/get-npm).

**Comment by iamericfletcher** [comment]
@RayBB Got it to work.  I will chip away at the errors and will work on my PR reviews. Thank you for helping.

**Comment by allanfinch34** [comment]
**Thank you for sharing excellent information. Your website is so cool. I am impressed by the details that you have on this website. It reveals how nicely you understand this subject. Bookmarked this website page, will come back for extra articles. You, my friend, ROCK! I found simply the info I already searched everywhere and simply could not come across. What a great website.** Visit@:-
[www.norton.com/setup](https://nortoncomblog.usl)
[www.office.com/setup](https://officecommerce.us)

**Comment by iamericfletcher** [comment]
Thank you @allanfinch34! If you enjoy the content, perhaps you can give it a star or a follow. I've corrected all the errors and am just waiting for the 30 day requirement and will continue working on PR. My goal is to have my list added and to also be a active contributor to this project!

**Comment by iamericfletcher** [comment]
unicorn.

**Comment by iamericfletcher** [comment]
@RayBB I have updated my list as well as this pull request. Can you please re-review it? Thank you!

The link in the files changed section should be replaced with the new address at the top of this pull request.

**Comment by iamericfletcher** [comment]
@RayBB Thank you for taking a look. 

According to the list requirements:

> Has been around for at least 30 days.
> That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.

The first real commit to my list occurred on July 30th, 2020. Wouldn't that satisfy the list requirement?

Also, I have added descriptions to the `YouTube` channels.

**Comment by iamericfletcher** [comment]
> Thanks for making an Awesome list! 🙌
> 
> It looks like you didn't read the [guidelines](https://github.com/sindresorhus/awesome/blob/master/pull_request_template.md) closely enough. I noticed multiple things that are not followed. Try going through the list point for point to ensure you follow it. I spent a lot of time creating the guidelines so I wouldn't have to comment on common mistakes, and rather spend my time improving Awesome.

@sindresorhus - Good afternoon, Sindre. I have implemented the changes suggested by the reviewers. They have been accepted, and am wondering if you have any further instruction?

**Review by iamericfletcher — COMMENTED** [review]
Should I close this PR?

**Comment by wernerkrauss** [comment]
Hi @iamericfletcher 
I wonder why "Resources" is a Heading2 and Blog etc. is a Heading3. This makes your Content section look, well, unusual.

Also the subheadings in Resources (Functions, Spatials etc.) are just bold text and not formatted as a heading.

The linter complains about line 109:
```
  ×  109:182  "’" is used without matching "‘"  remark-lint:match-punctuation
  ×  109:225  "’" is used without matching "‘"  remark-lint:match-punctuation
```

it seems you didn't use the apostrophe `'` but a `’` in O'Reilly and Shiny's.

The logo doesn't seem to be hi-res?

The descriptions could be improved. e.g in the YouTube section, the channel name is redundant:

> [Andrew Couch]() - YouTube channel created by Andrew Couch. Topics include modeling, creating functions, dashboards, and forecasting.

The "YouTube channel created by Andrew Couch" brings no value.

Please don't close this PR as your list is already pretty awesome.

**Comment by iamericfletcher** [comment]
> Hi @iamericfletcher
> I wonder why "Resources" is a Heading2 and Blog etc. is a Heading3. This makes your Content section look, well, unusual.
> 
> Also the subheadings in Resources (Functions, Spatials etc.) are just bold text and not formatted as a heading.
> 
> The linter complains about line 109:
> 
> ```
>   ×  109:182  "’" is used without matching "‘"  remark-lint:match-punctuation
>   ×  109:225  "’" is used without matching "‘"  remark-lint:match-punctuation
> ```
> 
> it seems you didn't use the apostrophe `'` but a `’` in O'Reilly and Shiny's.
> 
> The logo doesn't seem to be hi-res?
> 
> The descriptions could be improved. e.g in the YouTube section, the channel name is redundant:
> 
> > Andrew Couch - YouTube channel created by Andrew Couch. Topics include modeling, creating functions, dashboards, and forecasting.
> 
> The "YouTube channel created by Andrew Couch" brings no value.
> 
> Please don't close this PR as your list is already pretty awesome.

@wernerkrauss - Thank you for the kind words! I have implemented your suggestions. The list looks much better! The issue with O'Reilly and Shiny's should be ignored. They are in fact proper apostrophes. As for the logo, I followed the instructions, so the image should be hi-res. I admit the logo needs to be worked on. Do you know anyone who could help with that? Some of the other lists have amazing logos that are quite similar to one another.

I would love to get my list added by @sindresorhus and will do whatever it takes to make that happen! 

Sincerely,

Eric

**Review by iamericfletcher — COMMENTED** [review]
Please note the link shown in the Files Changed section is outdated:

The new link is:

[https://github.com/iamericfletcher/awesome-r-learning-resources](https://github.com/iamericfletcher/awesome-r-learning-resources)

File: readme.md [hunk 1]
```diff
@@ -162,7 +162,8 @@
 - [Lua](https://github.com/LewisJEllis/awesome-lua#readme)
 - [C](https://github.com/aleksandar-todorovic/awesome-c#readme)
 - [C/C++](https://github.com/fffaraz/awesome-cpp#readme) - General-purpose language with a bias toward system programming and embedded, resource-constrained software.
-- [R](https://github.com/qinwf/awesome-R#readme)
+- [R](https://github.com/qinwf/awesome-R#readme) - Functional programming language and environment for statistical computing and graphics.
+	- [Additional Learning Resources](https://github.com/iamericfletcher/R-Learning-Resources)
```
Comment (sindresorhus) [inline readme.md:166]: Should be just `Learning` and you forgot `#readme`.

Comment (iamericfletcher) [reply readme.md:166]: Hi @sindresorhus - This PR was orphaned by mistake (still learning my way around GitHub). I created a new PR that accounts for your commented items. 

[Pull Request #1882](https://github.com/sindresorhus/awesome/pull/1882)

**Review by iamericfletcher — COMMENTED** [review]
Hi @sindresorhus - This PR was orphaned by mistake (still learning my way around GitHub). I created a new PR that accounts for your commented items.

Pull Request #1882
